Failed nose import in test_annotation_update

Testing current master at 4d1d12c39 :

```
____________________________ test_annotation_update ____________________________
[gw0] linux -- Python 3.7.0 /venv/bin/python3.7
    def test_annotation_update():
        fig, ax = plt.subplots(1, 1)
        an = ax.annotate('annotation', xy=(0.5, 0.5))
        extent1 = an.get_window_extent(fig.canvas.get_renderer())
        fig.tight_layout()
        extent2 = an.get_window_extent(fig.canvas.get_renderer())
    
        np.testing.assert_raises(AssertionError, np.testing.assert_allclose,
                                 extent1.get_points(), extent2.get_points(),
>                                rtol=1e-6)
/venv/lib/python3.7/site-packages/matplotlib/tests/test_text.py:513: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
    def import_nose():
        """ Import nose only when needed.
        """
        nose_is_good = True
        minimum_nose_version = (1, 0, 0)
        try:
            import nose
        except ImportError:
            nose_is_good = False
        else:
            if nose.__versioninfo__ < minimum_nose_version:
                nose_is_good = False
    
        if not nose_is_good:
            msg = ('Need nose >= %d.%d.%d for tests - see '
                   'http://nose.readthedocs.io' %
                   minimum_nose_version)
>           raise ImportError(msg)
E           ImportError: Need nose >= 1.0.0 for tests - see http://nose.readthedocs.io
/venv/lib/python3.7/site-packages/numpy/testing/nose_tools/utils.py:71: ImportError
```

https://travis-ci.org/MacPython/matplotlib-wheels/jobs/414822234

--- a/lib/matplotlib/tests/test_text.py
+++ b/lib/matplotlib/tests/test_text.py
@@ -508,6 +508,5 @@ def test_annotation_update():
     fig.tight_layout()
     extent2 = an.get_window_extent(fig.canvas.get_renderer())
 
-    np.testing.assert_raises(AssertionError, np.testing.assert_allclose,
-                             extent1.get_points(), extent2.get_points(),
-                             rtol=1e-6)
+    assert not np.allclose(extent1.get_points(), extent2.get_points(),
+                           rtol=1e-6)

